src/fides/api/schemas/privacy_request.py

```diff
@@ -339,3 +340,42 @@ class RequestTaskCallbackRequest(FidesSchema):
     rows_masked: Optional[int] = Field(
         default=None, description="Number of records masked, as an integer"
     )
+
+
+class PrivacyRequestFilter(FidesSchema):
+    request_id: Optional[str] = None
+    identities: Optional[Dict[str, Any]] = Field(
+        None, example={"email": "user@example.com", "loyalty_id": "CH-1"}
+    )
+    custom_privacy_request_fields: Optional[Dict[str, Any]] = Field(
+        None, example={"site_id": "abc", "subscriber_id": "123"}
+    )
+    status: Optional[Union[PrivacyRequestStatus, List[PrivacyRequestStatus]]] = None
+    created_lt: Optional[datetime] = None
+    created_gt: Optional[datetime] = None
+    started_lt: Optional[datetime] = None
+    started_gt: Optional[datetime] = None
+    completed_lt: Optional[datetime] = None
+    completed_gt: Optional[datetime] = None
+    errored_lt: Optional[datetime] = None
+    errored_gt: Optional[datetime] = None
+    external_id: Optional[str] = None
+    action_type: Optional[ActionType] = None
+    verbose: Optional[bool] = False
+    include_identities: Optional[bool] = False
+    include_custom_privacy_request_fields: Optional[bool] = False
```

Comment:
Some of these fields felt strange to me at first being in the request body instead of being query params, but with the introduction of these more complex searches like custom_privacy_request_fields, which probably do make sense here, I agree with the call to just keep them all together in one place -